stablehlo/conversions/tosa/tests/quantToStablehlo.mlir

```diff
@@ -0,0 +1,153 @@
+// RUN: stablehlo-opt %s --stablehlo-quant-legalize-to-tosa-rescale --tosa-rescale-legalize-to-stablehlo --split-input-file -verify-each | FileCheck %s
```

Comment:
IMO, standalone testing of the pass pipeline  will not add much value as the passes are tested in isolation. However, would be worth exploring the rescale lowering pass in the context of interpreter. This fits perfectly into one of our objective of interpreting the output of `stablehlo-legalize-quant-to-tosa-rescale`.

How about we create a file `stablehlo/tests/interpret/tosa_rescale.mlir` and populate that with the output of `stablehlo-opt  --stablehlo-legalize-quant-to-tosa-rescale` and interpreter using 

```
// RUN: stablehlo-opt --tosa-legalize-rescale-to-math %s | stablehlo-translate --interpret
```

While populating the file, we may choose any one of the ops (add or subtract) with the goal to test just the rescale operations.

Reply:
> However, would be worth exploring the rescale lowering pass in the context of interpreter.

+1 to this, but let's keep the test file in `tosa/tests`, in terms of a component this would fall under code that TOSA devs are most knowledgeable on, so keeping the associated tests there will make for easier maintenance.

I'd recommend we start with `tosa/tests/rescale_interpreter.mlir`, and if we end up needing multiple test files we can make a new `tosa/tests/interpret/` directory, but for a single file we can hold off.

Reply:
done